rsp/runtimes/bundles/org.jboss.tools.rsp.server.tomcat/src/main/resources/servers.json

```diff
@@ -283,7 +291,7 @@
 							"workingDirectory": "bin",
 							"mainType": "org.apache.catalina.startup.Bootstrap",
 							"programArgs": "stop",
-							"vmArgs": "-Djava.util.logging.config.file=${server.home.dir}/conf/logging.properties -Djava.util.logging.manager=org.apache.juli.ClassLoaderLogManager -Djdk.tls.ephemeralDHKeySize=2048 -Djava.protocol.handler.pkgs=org.apache.catalina.webresources -Dorg.apache.catalina.security.SecurityListener.UMASK=0027 -Dcatalina.base=${server.home.dir} -Dcatalina.home=${server.home.dir} -Djava.io.tmpdir=${server.home.dir}/temp",
+							"vmArgs": "-Djava.util.logging.config.file=${server.home.dir}/conf/logging.properties -Djava.util.logging.manager=org.apache.juli.ClassLoaderLogManager -Djdk.tls.ephemeralDHKeySize=2048 -Djava.protocol.handler.pkgs=org.apache.catalina.webresources -Dorg.apache.catalina.security.SecurityListener.UMASK=0027 -Dcatalina.base=${server.base.dir} -Dcatalina.home=${server.home.dir} -Djava.io.tmpdir=${server.home.dir}/temp",
```

Comment:
```suggestion
							"vmArgs": "-Djava.util.logging.config.file=${server.base.dir}/conf/logging.properties -Djava.util.logging.manager=org.apache.juli.ClassLoaderLogManager -Djdk.tls.ephemeralDHKeySize=2048 -Djava.protocol.handler.pkgs=org.apache.catalina.webresources -Dorg.apache.catalina.security.SecurityListener.UMASK=0027 -Dcatalina.base=${server.base.dir} -Dcatalina.home=${server.home.dir} -Djava.io.tmpdir=${server.base.dir}/temp",
```

According to the Tomcat Official Document, the folder structure is recommanded like below:

![image](https://user-images.githubusercontent.com/33516065/164219341-9a84f4f6-116f-4fe9-b2fc-0c3336206c94.png)

So, I thought the `-Djava.util.logging.config.file` and `-Djava.io.tmpdir` should pointed to ${server.base.dir} but not ${server.home.dir}